modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in the target interface definition of a machine config manifest file.  
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
+
+* Drains nodes in singular order based on the selected machine config pool.
+* Injects Ignition configuration files into each node, so that each node receives the additional `br-ex` bridge network configuration.
+* Restarts the node with the modified override file.
+* Executes the `configure-ovs.sh` shell script that references the new interface definition.
+* Adds `br-ex` and `br-ex1` to the host node.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology".
+
+.Procedure
+
+. Convert the rule to Base64 encoded by running the following command:
```

Comment:
For this section, I think we will want to establish a brief statement about what we're doing first, before we get to steps:

Suggested:

~~~
Optional: Create an interface connection that your secondary `br-ex1` interface will utilize by following steps below. Here exampled as a configuration to define a secondary bond interface using two ifaces that are currently not utilized by the cluster via machine-configuration tasks.
~~~

The [Important] Block is good, but should be moved up under this subheading before step 1. This way our customers understand that if they have an existing interface to use, they can skip these steps, if they don't they must take action to ensure a connection exists prior to the next parts of the tasking. 


Step 1 here: `Convert the rule to Base64 encoded by running the following command` is accurate, but needs to come after the definition of said file. Add a block with example iface definition for eno1 and a second one for eno2, and a third file with your bond interface `bond1` which will be used to host `br-ex1`

~~~
. Create the following interface definition files which will be embedded in a machine-configuration for placement on your host nodes. Note that the below details are provided as a baseline example/template and your specific network configuration may differ or have varying requirements for your network.
~~~

$ cat eno1.config

~~~
[connection]
id=eno1
type=ethernet
interface-name=eno1
master=bond0
slave-type=bond
autoconnect=true
autoconnect-priority=20
~~~

$ cat eno2.config

~~~
[connection]
id=eno2
type=ethernet
interface-name=eno2
master=bond0
slave-type=bond
autoconnect=true
autoconnect-priority=20
~~~

$cat bond1.config

~~~
[connection]
id=bond1
type=bond
interface-name=bond1
autoconnect=true
connection.autoconnect-slaves=1
autoconnect-priority=20

[bond]
mode=802.3ad
miimon=100
xmit_hash_policy="layer3+4"

[ipv4]
method=auto
~~~

Followed by:

~~~
. Convert the files to Base64 encoded strings by running the following commands:

[source,terminal]
----
$ base64 /path/to/en01.config
$ base64 /path/to/eno2.config
$ base64 /path/to/bond1.config
----
~~~

*NOTE: this is assuming we want to give an example of a bonded interface here. We could always simplify by just having them define `eno2` for their config instead of a bonded iface but this is more thorough.

Reply:
:heavy_check_mark: